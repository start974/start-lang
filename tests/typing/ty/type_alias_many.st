type N1 := N
type Na := N
type N2 := N1
type N3 := N2
type N4 := N3

def a := 1
def b : N4 := a

