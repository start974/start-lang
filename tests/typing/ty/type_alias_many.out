def a : N := 1
def b : N4 := a
